Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3240,6 +3241,39 @@ CSSRule* Parser::convert_to_rule(NonnullRefPtr<Rule> rule)
 
             return CSSKeyframesRule::create(m_context.realm(), name, move(keyframes)).release_value_but_fixme_should_propagate_errors();
         }
+        if (rule->at_rule_name().equals_ignoring_ascii_case("namespace"sv)) {
+            // https://drafts.csswg.org/css-namespaces/#syntax
+            auto token_stream = TokenStream { rule->prelude() };
+            dbgln("Parsing namespace");
+            auto transaction = token_stream.begin_transaction();
```

Comment:
This transaction isn't useful, because the `TokenStream` is already temporary inside this block.